x/datadeal/types/keys.go

```diff
@@ -23,8 +25,14 @@ const (
 
 var (
 	KeyIndexSeparator = []byte{0xFF}
+
+	DataSaleKey = []byte{0x02}
 )
 
+func GetDataSaleKey(verifiableCID []byte, dealID uint64) []byte {
+	return append(DataSaleKey, CombineKeys(verifiableCID, sdk.Uint64ToBigEndian(dealID))...)
```

Comment:
IMO, it would be better to change the order, which will enable to get the data sales of a specific deal.
```suggestion
	return append(DataSaleKey, CombineKeys(sdk.Uint64ToBigEndian(dealID), verifiableCID)...)
```

Reply:
Fixed in [254c02d](https://github.com/medibloc/panacea-core/pull/421/commits/254c02dfcb14b3100c057fcacbf3d0ac25f60b46). And I also change convert method about byte array to string in `keys.go`.